storybook/stories/core/components/Input/DropdownInput.stories.js

```diff
@@ -0,0 +1,118 @@
+import DropdownInput from '@orangehrm/oxd/core/components/Input/DropdownInput';
+
+export default {
+  title: 'Example/DropdownInput',
+  component: DropdownInput,
+  argTypes: {
+    style: {control: {type: 'object'}},
+    hasError: {control: {type: 'boolean'}},
+  },
+};
+
+const Template = args => ({
+  setup() {
+    return {args};
+  },
+  components: {'oxd-dropdown-input': DropdownInput},
+  template: '<oxd-dropdown-input v-bind="args" />',
+});
+
+const dropdownOptions = [
+  {
+    id: 1,
+    label: 'HR Admin',
+  },
+  {
+    id: 2,
+    label: 'ESS User',
+  },
+  {
+    id: 3,
+    label: 'Supervisor',
+  },
+];
+
+const syncFunction = function(serachParam) {
+  console.log(serachParam);
+  return dropdownOptions;
+};
+
+const asyncFunction = async function(serachParam) {
+  console.log(serachParam);
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(dropdownOptions);
+    }, 5000);
+  });
+};
+
+const asyncLazyFunction = async function(serachParam) {
+  return new Promise(resolve => {
+    if (serachParam.trim()) {
+      console.log(serachParam);
```

Comment:
remove `console.log(serachParam);` here and above if unnecessary

Reply:
updated and removed.